ID3TagEditor.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
+	<true/>
+</dict>
+</plist>
```

Comment:
This file could be removed.

Reply:
I don't think I created this one? Either it was generated automatically, or it's one of yours, which generally I didn't touch. But I'll remove it.

Reply:
Actually I can't even find this to remove it. It must be something that was generated automatically, and at my current skill level, I don't trust myself enough to mess with things I don't understand.

Reply:
No worries, I will remove it after we merge the PR (before releasing a new version) 👍.